settings.gradle

```diff
@@ -18,3 +18,4 @@ include ':nostrpostrlib'
 include ':bip-schnorr-kotlin'
 include ':examples'
 include ':NostrRelay'
+include ':nostrpostr-android'
```

Comment:
Why did you add this? It's basically empty? Why not use `app` that is there already? Please remove it or use it but don't just drop dead code in the project.